fixups/RegLegacyFixups/dllmain.cpp

```diff
@@ -19,12 +18,35 @@ bool trace_function_entry = false;
 bool m_inhibitOutput = false;
 bool m_shouldLog = true;
 
-void InitializeFixups();
```

Comment:
I am assuming the log removed here is not being used for any BI dashboards internally - Please confirm on the same.

Reply:
Yes the logs are not related to BI dashboards at all